vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/validation/BinderValidationIT.java

```diff
@@ -58,17 +59,17 @@ public void fieldIsInitiallyValid() {
     public void required_triggerDateInputBlur_assertValidity() {
         dateInput.sendKeys(Keys.TAB);
         timeInput.sendKeys(Keys.TAB);
-        assertServerInvalid();
-        assertClientInvalid();
-        assertErrorMessage(REQUIRED_ERROR_MESSAGE);
+        assertServerValid();
+        assertClientValid();
+        assertErrorMessage(null);
```

Comment:
Each test case should also assert the validation count.

Reply:
Added count assertions.